awx/ui_next/src/screens/Project/shared/ProjectForm.jsx

```diff
@@ -164,10 +160,11 @@ function ProjectFormFields({
         isValid={!organizationMeta.touched || !organizationMeta.error}
         onBlur={() => organizationHelpers.setTouched()}
         onChange={value => {
-          organizationHelpers.setValue(value.id);
-          setOrganization(value);
+          organizationHelpers.setValue(value);
         }}
-        value={organization}
+        value={organizationField.value}
+        touched={organizationMeta.touched}
+        error={organizationMeta.error}
```

Comment:
Do we need to add these two props? It doesn't look like the OrganizationLookup uses touched or error, but I've seen this pattern in a few other places.

Reply:
@jlmitch5 maybe you can help shed some light on these props.  I followed patterns that I saw elsewhere with respect to configuring lookups but it doesn't look like `touched` or `error` is consumed by the lookup(s).  Should we remove these props or should we add logic to the lookups to consume them in some way?

Reply:
@marshmalien @mabashian when i was converting things from using Field -> useField, I was doing it based 100% on what was happening (i.e. being passed) before.  I didn't go in and check or change how they were be using.  It's probably that we got rid of the used of the touched/error props sometime, but didn't remove the props here.

Reply:
>we got rid of the used of the touched/error props sometime, but didn't remove the props here

@mabashian @jlmitch5 Sounds like we can remove these props 👍